ecosystem/utils/utils.py

```diff
@@ -37,4 +38,8 @@ def set_actions_output(outputs: List[Tuple[str, Union[str, bool, float, int]]])
     """
     for name, value in outputs:
         logger.info("Setting output variable %s: %s", name, value)
-        print("::set-output name={name}::{value}".format(name=name, value=value))
+        if "GITHUB_OUTPUT" in os.environ:
```

Comment:
When would this not be true? It sounds like this script is only meant for CI. I'd think we want to raise an `AssertionError` if it's not set.

Reply:
During unit test in local you don't have this variable and so I unset it for the tox

Reply:
> and to keep a test for the output.

Fair. What about then keeping this as is, but changing this if statement to check for `CI` rather than `GITHUB_OUTPUT`? That will make this code more self-documenting. 

Maybe add a brief comment to the `else` branch that we still print in order to test?

Reply:
Yes go for this one :)
It's gonna be more clear indeed.

TODO :
- [ ] Change `GITHUB_OUTPUT` to `CI`
- [ ] Add comment into `else` branch
- [ ] Ignore `CI` in `tox.ini`

I'll do it later in the day